src/TConsole.cpp

```diff
@@ -346,17 +346,20 @@ TConsole::TConsole(Host* pH, bool isDebugConsole, QWidget* parent)
     layoutLayer2->setMargin(0);
     layoutLayer2->setSpacing(0);
 
-    auto buttonMainLayer = new QWidget; //( layerCommandLine );
+    buttonMainLayer = new QWidget; //( layerCommandLine );
```

Comment:
Is there any point to renaming the pointer to `mpButtonMainLayer` it helps sometimes to work out which variables are class members and which are automatics - despite the extra work if one changes into the other like here - so as a reminder that it ought to be nulled in the initialization list for instance?

Reply:
Done.